workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/container/AndroidDialogBounds.kt

```diff
@@ -0,0 +1,77 @@
+package com.squareup.workflow1.ui.container
+
+import android.app.Dialog
+import android.view.Gravity
+import android.view.View
+import android.view.Window
+import android.view.WindowManager
+import android.view.WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL
+import com.squareup.workflow1.ui.ViewEnvironment
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.environment
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+
+@WorkflowUiExperimentalApi
+public fun Dialog.setBounds(bounds: Bounds) {
+  window?.let { window ->
+    val windowRect = WindowManager.LayoutParams()
+      .apply {
+        copyFrom(window.attributes)
```

Comment:
Also you can reassign the same attributes. I can confirm it works:
`attributes = attributes.apply { ... }`
I know it might be smelly but...

Reply:
I'll hold my nose to avoid allocs.